app/views/unattended/report_templates/host_-_applicable_errata.erb

```diff
@@ -16,23 +16,31 @@ template_inputs:
   description: Limit the report only on errata found by this search query. Keep empty
     for report on all available errata.
   advanced: false
+- name: Installability
+  required: true
+  input_type: user
+  description: Show applicable or installable errata?
```

Comment:
Should maybe have a more explanatory tooltip here?

```suggestion
  description: Select Applicable to show all applicable errata. Select Installable to restrict the report only to errata that are actually available in the host's content view environment(s) and can actually be installed.
```

Reply:
Looks like this was updated in the other report but not here, was that intentional?

Reply:
Oops, I'll fix that.